vue/dist/LogViewer.umd.js.map

```diff
@@ -1 +1 @@
hText(text: string) {\n      if (!text) {\n        return;\n      }\n\n      this.selectedSeverity = '';\n      this.query = text;\n\n      if (this.useRegExp) {\n        this.query = window.preg_quote(this.query);\n      }\n\n      this.update();\n    },\n    previousPage() {\n      if (this.page > 0) {\n        this.page -= 1;\n        this.update();\n      }\n    },\n    nextPage() {\n      if (this.logs?.length) {\n        this.page += 1;\n        this.update();\n      }\n    },\n    onSearchKeydown(event: KeyboardEvent) {\n      if (event.code !== 'Enter') {\n        return;\n      }\n\n      this.update();\n    },\n  },\n  computed: {\n    buildQuery() {\n      let { query } = this;\n\n      if (!this.useRegExp) {\n        query = window.preg_quote(this.query);\n      }\n\n      if (this.selectedSeverity) {\n        query = `^${this.selectedSeverity}(.*)${query}`;\n      }\n\n      return query;\n    },\n    exportFormAction() {\n      return `?${MatomoUrl.stringify({\n        module: 'API',\n        method: 'LogViewer.getLogEntries',\n        query: this.buildQuery,\n        format: 'TSV',\n        page: 0,\n        limitPerPage: 5000,\n      })}`;\n    },\n    logConfigJson() {\n      return JSON.stringify(this.logConfig || {});\n    },\n    tokenAuth() {\n      return Matomo.token_auth;\n    },\n    logsWithMessage() {\n      return (this.logs || []).filter((l) => l.message);\n    },\n  },\n});\n","export { default } from \"-!../../../../../node_modules/@vue/cli-plugin-typescript/node_modules/cache-loader/dist/cjs.js??ref--14-0!../../../../../node_modules/babel-loader/lib/index.js!../../../../../node_modules/@vue/cli-plugin-typescript/node_modules/ts-loader/index.js??ref--14-2!../../../../../node_modules/@vue/cli-service/node_modules/cache-loader/dist/cjs.js??ref--0-0!../../../../../node_modules/@vue/cli-service/node_modules/vue-loader-v16/dist/index.js??ref--0-1!./LogViewer.vue?vue&type=script&lang=ts\"; export * from \"-!../../../../../node_modules/@vue/cli-plugin-typescript/node_modules/cache-loader/dist/cjs.js??ref--14-0!../../../../../node_modules/babel-loader/lib/index.js!../../../../../node_modules/@vue/cli-plugin-typescript/node_modules/ts-loader/index.js??ref--14-2!../../../../../node_modules/@vue/cli-service/node_modules/cache-loader/dist/cjs.js??ref--0-0!../../../../../node_modules/@vue/cli-service/node_modules/vue-loader-v16/dist/index.js??ref--0-1!./LogViewer.vue?vue&type=script&lang=ts\"","import { render } from \"./LogViewer.vue?vue&type=template&id=f03f9e24\"\nimport script from \"./LogViewer.vue?vue&type=script&lang=ts\"\nexport * from \"./LogViewer.vue?vue&type=script&lang=ts\"\nscript.render = render\n\nexport default script","/*!\n * Matomo - free/libre analytics platform\n *\n * @link https://matomo.org\n * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later\n */\n\nexport { default as LogViewer } from './LogViewer/LogViewer.vue';\n","import './setPublicPath'\nexport * from '~entry'\n"],"sourceRoot":""}
```

Comment:
@diosmosis Are those `*.map` files added on purpose?

Reply:
No, thanks for catching that